backend/src/routes/user.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * This class contains routes that will create and get
+ * new users
+ */
+import express, { NextFunction, Request, Response } from "express";
+import { Mentee } from "../models/mentee";
+import { Mentor } from "../models/mentor";
+import { validateMentee, validateMentor } from "../middleware/validation";
+import { createUser } from "../services/auth";
+
+const router = express.Router();
+
+/**
+ * This is a post route to create a new mentee. It will first validate
+ * the mentee to ensure they follow standard listed below before adding them to
+ * mongoDB and Firebase
+ *
+ * Mentee: {type: string, name: string, email: string,password: string}
+ */
+router.post(
+  "/mentee",
+  [validateMentee],
+  async (req: Request, res: Response, next: NextFunction) => {
+    console.log("Creating a new mentee", req.query);
+
+    try {
+      const { name, email, password } = req.body;
+      const status = "under review";
+      const mentee = new Mentee({ name, status });
+      await createUser(mentee._id.toString(), email, password);
+      await mentee.save();
+      return res.status(201).json({
+        message: `Mentee ${name} was succesfully created.`,
+        userID: mentee._id,
+      });
+    } catch (e) {
+      next();
+      return res.status(400);
+    }
+  }
+);
+
+/**
+ * This is a post route to create a new mentor. It will first validate
+ * the mentor to ensure they follow standard listed below before adding them to
+ * mongoDB and Firebase
+ *
+ * Mentor: {type: string, name: string, email: string, password: string
+ * organization_id: string, personal_access_token: string}
+ */
+
+router.post(
```

Comment:
Similar message for here. Check this figma for missing fields on this route.
<img width="177" alt="image" src="https://user-images.githubusercontent.com/56745367/219135882-a6fe6e12-e7c3-461d-b723-bf3e0450714b.png">

Reply:
https://github.com/TritonSE/ALUM-Mobile-Application/issues/46